src/Application/Migrations/24_function_ticket_reset.sql

```diff
@@ -0,0 +1,75 @@
+BEGIN;
+
```

Comment:
Also, it should be documented, that this function resets handle_id and failed flag additionally to resetting the state.